### PR TITLE
Adding docs page for eCloud VPC VPN's

### PR DESCRIPTION
**Pull request by hep147258**

### CODE REVIEW

**Review by CountBacula — APPROVED** [review]
LGTM

File: source/ecloud/ecloud-vpc/vpn.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+A "Pre-Shared Key (PSK)" needs to be defined, which will need to match the PSK on the remote device.
+
+![VPN Session](files/vpn-session.png)
+
+
+#### Profile Groups
+
+There have been a range of VPN Profiles created, which have various levels of encryption settings. Details of these can be seen in the following table. If you need any advise on which VPN profiles to use, please get in touch with our support team.
+
+| Profile Name        | IKE Version | Phase 1 - Encryption | Phase 1 - Integrity | Phase 1 - DH Group | Phase 1 - PRF | Phase 1 - SA Lifetime | Phase 2 - Encryption | Phase 2 - Integrity | Phase 2 - SA Lifetime | Phase 2 - PFS |
```
Comment (Xiol) [inline source/ecloud/ecloud-vpc/vpn.md]: This table is broken when rendered.

See https://pr-1002.docsuat.devops.ans.uk/ecloud/ecloud-vpc/vpn.html

File: source/ecloud/ecloud-vpc/vpn.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# eCloud VPC VPNs
```
Comment (Xiol) [inline source/ecloud/ecloud-vpc/vpn.md]: This page isn't linked to the TOC for eCloud VPC, you'll need to add it in [here](https://github.com/ukfast/docs.ukfast.co.uk/blob/master/source/ecloud/ecloud-vpc/index.rst).

Comment (hep147258) [reply source/ecloud/ecloud-vpc/vpn.md]: Thanks @Xiol, I have added that now

File: source/ecloud/ecloud-vpc/vpn.md (hunk 1 above)
Comment (hep147258) [reply source/ecloud/ecloud-vpc/vpn.md]: Will have a look at this and see how best to do it

**Comment by miff2000** [comment]
I'm going to sort the table formatting out on this once and for all. Give me a little time 😊

**Review by miff2000 — APPROVED** [review]

**Pull request merged**